pkg/noderesourcetopology/filter.go

```diff
@@ -51,33 +47,12 @@ type PolicyHandlerMap map[topologyv1alpha1.TopologyManagerPolicy]PolicyHandler
 // TopologyMatch plugin which run simplified version of TopologyManager's admit handler
 type TopologyMatch struct {
 	policyHandlers PolicyHandlerMap
-	lister         listerv1alpha1.NodeResourceTopologyLister
-	namespaces     []string
+	data           commonPluginsData
```

Comment:
data as a field looks odd, here and in other places where commonPluginData is used, I think embed type much better fits our needs.
Unless you have a point to use yet another level of aggregation.